.gitignore

```diff
@@ -33,6 +33,7 @@ share/python-wheels/
 .installed.cfg
 *.egg
 MANIFEST
+/.deps/
```

Comment:
Does the leading slash make the path to root?

Reply:
It anchors the path to the repository's root (not the machine's root)